### ISSUE TITLE
ACL Action Restriction for Downtime Page: Cancel button not hidden

### ISSUE THREAD

**Issue by Iralein** [centreon/centreon#6466]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 
Centreon 2.8.23

**Steps to reproduce the issue:**
1. Create a restricted user who have access to some services
2. Action ACL: no "set downtime"
3. Set downtime for the service the user has access to
3. Logged in as this user: go to Downtime page: You'll see the Cancel Button. If you use it, the popup is empty and nothing happens.

**Describe the results you received:**
The user can see the Cancel button. If he want's to cancel a downtime, he'll get an empty popup. Nothing happens.

**Describe the results you expected:**
As the configured user has no right to set downtime, he also should not see the Cancel button. It should be configurable by Action ACL.

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by Iralein** [comment]
![unbenannt](https://user-images.githubusercontent.com/13367814/42749808-473f540e-88e5-11e8-95f3-03736ab1e0c8.PNG)

**Comment by lpinsivy** [comment]
Hi @Iralein can you try this fix?

```
--- a/usr/share/centreon/www/include/monitoring/downtime/template/listDowntime.ihtml	2018-08-21 16:39:38.095074387 +0200
+++ b/usr/share/centreon/www/include/monitoring/downtime/template/listDowntime.ihtml	2018-08-21 16:46:55.059048341 +0200
@@ -30,7 +30,7 @@
 	<tr class="ToolbarTR">
         <td>
             {if $msgs2}<a class="btc bt_success" href="{$msgs2.addL2}">{$msgs2.addT2}</a>{/if}
-            {if $nb_downtime_svc}<input type="submit" name="submit2" value="{$cancel}" class="btc bt_danger" onclick="doAction('select[name=\'o1\']', 'cs','{$msgs2.delConfirm}')">{/if}
+            {if $nb_downtime_svc}{if $msgs2}<input type="submit" name="submit2" value="{$cancel}" class="btc bt_danger" onclick="doAction('select[name=\'o1\']', 'cs','{$msgs2.delConfirm}')">{/if}{/if}
         </td>
         {php}
         include('./include/common/pagination.php');
@@ -106,7 +106,7 @@
     <tr class="ToolbarTR">
         <td>
             {if $msgs2}<a class="btc bt_success" href="{$msgs2.addL2}">{$msgs2.addT2}</a>{/if}
-            {if $nb_downtime_svc}<input type="submit" name="submit2" value="{$cancel}" class="btc bt_danger" onclick="doAction('select[name=\'o1\']', 'cs','{$msgs2.delConfirm}')">{/if}
+            {if $nb_downtime_svc}{if $msgs2}<input type="submit" name="submit2" value="{$cancel}" class="btc bt_danger" onclick="doAction('select[name=\'o1\']', 'cs','{$msgs2.delConfirm}')">{/if}{/if}
         </td>
 	{php}
 	   include('./include/common/pagination.php');
```

**Comment by Iralein** [comment]
Hi,

The button "Cancel" has now disappeared, so, the issue is solved for me.
Maybe one should think about to hide the checkboxes, too, as they aren't usable for those users.
THX for help and kind regards,

Iralein

**Comment by Iralein** [comment]
BTW: the comment site hits the same issue, solution is pretty much the same.

**Comment by Iralein** [comment]
For 18.10, the bug is fixed.

**Comment by adr-mo** [comment]
If the issue has been fixed in 18.10 then **closing this issue**

Regards,
